internal/executable/executable.go

```diff
@@ -0,0 +1,45 @@
+package executable
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+var runmePath string
```

Comment:
More error-prone approach might be to define a function that creates a dump env command and replace it when running tests.

```go
// In internal/runner.
var DumpCmd string = getDumpCmd

func getDumpCmd() string {
    path, _ := os.Executable()
    ...
    // return `/path/to/runme env dump`
}

// In internal/runner/*_test.go
func init() {
  DumpCmd = `echo "STUB_ENV1=val1\x00STUB_ENV2=val2"`
}
```

Reply:
Yep, that's a significantly better way to do it.